gcloud/logging/handlers/transports/test_sync.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python
+# Copyright 2016 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import logging
+
+import unittest2
+
+
+class TestSyncHandler(unittest2.TestCase):
+
+    PROJECT = 'PROJECT'
+
+    def _getTargetClass(self):
+        from gcloud.logging.handlers.transports import SyncTransport
+        return SyncTransport
+
+    def _makeOne(self, *args, **kw):
+        return self._getTargetClass()(*args, **kw)
+
+    def test_ctor(self):
+        client = _Client(self.PROJECT)
+        NAME = "python_logger"
+        transport = self._makeOne(client, NAME)
+        self.assertEqual(transport.logger.name, "python_logger")
+
+    def test_send(self):
+        client = _Client(self.PROJECT)
+        STACKDRIVER_LOGGER_NAME = "python"
+        PYTHON_LOGGER_NAME = "mylogger"
+        transport = self._makeOne(client, STACKDRIVER_LOGGER_NAME)
+        MESSAGE = "hello world"
+        record = _Record(PYTHON_LOGGER_NAME, logging.INFO, MESSAGE)
+
+        transport.send(record, MESSAGE)
+        self.assertEqual(transport.logger.log_struct_called_with,
+                         ({
```

Comment:
nitpick, but the indentation seems a little funny here. I know it passes lint but this could be moved up, since the closing brace is inline?